changelog.txt

```diff
@@ -1,13 +1,16 @@
 *** WooCommerce Shipping & Tax Changelog ***
 
-= 1.25.10 - 2021-xx-xx =
-* Tweak - Removes deprecated Jetpack constant JETPACK_MASTER_USER
```

Comment:
Looks like this went away? 👀 I believe it should be part of `.11`, since I don't see it in `master`

Reply:
That was released as part of `1.25.9` https://github.com/Automattic/woocommerce-services/blob/f897309ea412a164be6460544f7e6dc9fc582edf/changelog.txt#L17 :smiley:

Reply:
Whoops, thanks! :D